src/data/general-cs-quiz.ts

```diff
@@ -72,6 +72,17 @@ const generalCSQuiz = [
     Explanation:
       "Big O notation describes the upper bound. The function n upper bounds the function 2*logn. However, it is not the tightest upper bound, and in general we always specify the tightest upper bound to quantify an algorithm.",
     Link: "https://www.youtube.com/watch?v=RBSGKlAvoiM&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=5m43s",
+  },
+   {
+    Question:
+      "Which is better Merge sort or Quicksort?",
+    Answer: "Merge sort",
+    Distractor1: "Merge sort",
+    Distractor2: "Quicksort",
+    Distractor3: "Both",
+    Explanation:
+      "Merge sort is more efficient and works faster than quick sort in case of larger array size or datasets.",
```

Comment:
Let's reword this because it is a direct quote from this article. Please avoid direct quotes from 3rd party articles. 
https://www.geeksforgeeks.org/quick-sort-vs-merge-sort

```suggestion
      "Merge sort runs quicker than Quick sort in the case of larger datasets.",
```